sites/public/src/components/account/StatusItem.tsx

```diff
@@ -23,20 +24,22 @@ interface StatusItemProps {
 
 const StatusItem = (props: StatusItemProps) => {
   return (
-    <article className={styles["status-item"]}>
-      <div className={styles["status-item__inner"]}>
+    <Card.Section
+      className={`${accountStyles["account-card-section"]} ${accountStyles["application-card-section"]}`}
+    >
+      <article className={styles["status-item"]}>
         <header className={styles["status-item__header"]}>
           <h3 className={styles["status-item__title"]}>{props.listingName}</h3>
-          {props.applicationDueDate && (
-            <p className={styles["status-item__due"]}>
-              {props.strings?.applicationDeadline ?? t("listings.applicationDeadline")}:{" "}
-              {props.applicationDueDate}
-            </p>
-          )}
+          <p className={styles["status-item__status"]}>
+            {props.strings?.status ?? t("application.status")}:{" "}
+            <Tag variant="primary">
+              {props.strings?.submittedStatus ?? t("application.statuses.submitted")}
```

Comment:
Was it not part of this ticket to have the new statuses rather than just always having "submitted"? https://github.com/metrotranscom/doorway/issues/434#issuecomment-1843353072. I think if I remember right they will be part of the side nav ticket, but just want to make sure it wasn't missed

Reply:
Yep, planning on covering those pieces in the side nav updates!